docs/en/Security.md

```diff
@@ -126,6 +127,57 @@ The owner, group, and permissions can be changed by two ways:
 The owner can only be changed by super user.
 The group and permission can only be changed by super user and file owner.
 
+## Access Control List
+Posix permission model allows administrators to grant permissions to owners, owning groups and other users. 
+The permission bits model is sufficient for most cases. 
+However, to help administrators express more complicated security policies,
+Alluxio also supports Access Control Lists (ACLs).
+ACLs allow administrators to grant permissions to any user or group. 
+
+In Alluxio's ACL model, a file or directory's Access Control List consists of many ACL entries. There are two types of ACL entries, Access ACL entries and Default ACL entries. 
+
+1. Access ACL Entries: 
+This type of ACL entry specifies a particular user or group's permission to read, write and execute. 
+Each ACL entry consists of a type, which can be one of user, group or mask, an optional name and a permission string similar to the POSIX permission bits. 
+The following table shows the different types of ACL entries that can appear in the access ACL. 
+
+	|ACL Entry Type| Description|
+|:------------------------:|:------------------------:|
+|  user:userid:permission  |  Sets the access ACLs for a user. Empty userid implies the permission is for the owner of the file.|
+| group:groupid:permission | Sets the access ACLs for a group. Empty groupid implies the permission is for the owning group of the file.|
+|     other::permission    | Sets the access ACLs for all users not specified above.|
+|     mask::permission     | Sets the effective rights mask.  The ACL mask indicates the maximum permissions allowed for all users other than the owner and for groups.|
```

Comment:
does the mask limit the maximum permissions for the group owning the object?

Reply:
Yes